aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Tagging.java

```diff
@@ -220,4 +221,9 @@ public static ErrorRuleSet bestEffortErrorRuleSet(
         }
         return hardFailErrorRuleSet;
     }
+
+    private static void addToMap(Map<String, Tag> allTags, Collection<Tag> tags){
+        for(Tag tag : tags)
+            allTags.put(tag.key(), tag);
```

Comment:
nit: Let's please curly-brace this block

Reply:
Done.